admin-ui.environment.yml.example

```diff
@@ -0,0 +1,28 @@
+---
+director_uuid: DIRECTOR_ID
+
+update:
```

Comment:
Yeah I think you're right. Per environment config isn't secret at all, and we just have `main` and `external` secrets. No need to keep update block in secrets either really.